src/app/routes/applications/new-role/components/role-name/role-name.component.ts

```diff
@@ -37,10 +37,9 @@ export class RoleNameComponent {
     return this.form.hasError(errorType);
   }
 
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
   alphaNumericOnly(event: any, includeDot?: boolean): boolean {
     this.existAndNotOwner = false;
-    return isAlphaNumericOnly(event, includeDot);
+    return true;
```

Comment:
Are you sure? :smile:

Reply:
Yeah, i should more focus on making it better than just fixing it. Sorry, i will rewrite more of it :)